website/src/components/Changelog/GUI.tsx

```diff
@@ -1,31 +1,12 @@
-import Link from "next/link";
 import Entry from "./Entry";
-import Entries from "./Entries";
+import Entries, { DownloadLink } from "./Entries";
 import ChangeItem from "./ChangeItem";
 import Unreleased from "./Unreleased";
+import { OS } from ".";
 
-export default function GUI({ title }: { title: string }) {
-  const downloadLinks =
-    title === "Windows GUI"
```

Comment:
This here is the bug. We are passing "Windows" as the title into this component.